.travis.yml

```diff
@@ -3,6 +3,7 @@ php:
   - '7.1'
```

Comment:
Since we are removing the compatibility with php 7.* are this and the following lines still necessary?

Reply:
Fixed.